### ISSUE TITLE
Description field for the templates gets truncated for long descriptions.

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#2356]
Server version - Build from master:

Description field for the templates gets truncated for long descriptions.

Is there limitation on how long the template description needs to be in catalog ?  We should be able to display the entire description in UI.

<img width="304" alt="screen shot 2015-10-16 at 11 54 30 am" src="https://cloud.githubusercontent.com/assets/4266958/10550642/e97b47b0-73fc-11e5-8cf7-bde292758fad.png">

**Comment by vincent99** [comment]
The boxes are fixed size.  The description should be shorter and we should have a separate longer readme that you can see for more info.

**Comment by westlywright** [comment]
I agree. It seems like we should have a "shortDescription" field and a way to view the full description. That would also pair with an API change. @vincent99 do you want me to open a ticket for this? do we need to get this one wrapped up before monday?

**Comment by vincent99** [comment]
This is the issue; I don't think it needs to be in for Monday.  Maybe @cloudnautique can just shorten the ElasticSearch one for now.

**Comment by cloudnautique** [comment]
Yeah, I'll shorten itshorten for now. Agree with the longer term solution when we can get to it. A "Readme" type of section would nice, and could fit into the description field. What is the character limit? If its longer then a tweet, Its too wordy :)

**Comment by vincent99** [comment]
Variable width font, but something on that order.  Maybe it could just not say "ElasticSearch" 5 times.

Also it seems like specific versions shouldn't be in the top-level description.  As soon as there's an update 0 and 1 will have different versions.

**Comment by vincent99** [comment]
All templates have short enough descriptions to fit now, and the readme can include more separately.
